docs/datamodel/access_policies.rst

```diff
@@ -205,28 +247,41 @@ Let's add a policy to our sample schema.
 
     +     access policy author_has_full_access
     +       allow all
-    +       using (global current_user ?= .author.id);
+    +       using (global current_user    ?= .author.id
+    +         and  global current_country ?= Country.Full) {
+    +        errmessage := "User not in country with full access";
+    +       }
+    +      access policy author_has_read_access
+    +        allow select
+    +        using (global current_user    ?= .author.id 
+    +          and  global current_country ?= Country.ReadOnly);
         }
 
-
-Let's break down the access policy syntax piece-by-piece. This policy grants
-full read-write access (``all``) to the ``author`` of each ``BlogPost``. No
-one else will be able to edit, delete, or view this post.
+Let's break down the access policy syntax piece-by-piece. These policies grant
+full read-write access (``all``) to the ``author`` of each ``BlogPost``, if
+the author is in a country that allows full access to the service. Otherwise,
+the same author will be restricted to either read-only access or no access at
+all, depending on the country.
 
 .. note::
 
   We're using the *coalescing equality* operator ``?=`` which returns
   ``false`` even if one of its arguments is an empty set.
 
-- ``access policy``: The keyword used to declare a policy inside an object
+- ``access policy`` The keyword used to declare a policy inside an object
```

Comment:
I'd vote for restoring the colon here. Maybe it was unintentional?
```suggestion
- ``access policy``: The keyword used to declare a policy inside an object
```

Reply:
Yes, I think I killed that comma unintentionally -_-